### DISCUSSION TITLE
Slow performance on specific terminal emulator

### DISCUSSION THREAD

**Discussion by jansen44** [helix-editor/helix#10203] [General]
So, I'm trying helix for the first time and felt it really slow, literally flickering while scrolling down a file holding "j". 
I tried using another terminal emulator and it just ran smoothly.

I don't know if it's a direct problem to Helix, but it's really annoying since the slow terminal emulator (Tabby) runs smoothly with every other program (including vim/nvim + plugins) and I really want some help with it.

I also don't know if it happens to another terminal emulators or if this was reported before.

I'll be attaching a little video of the situation.

OS: MacOs 12.5
Machine Specs: Macbook Pro 13 2017
Slow Terminal Emulator: Tabby
Normal Terminal Emulator: MacOS Terminal

Edit: I tagged this as "enhancement" because I didn't know if this could be considered a bug.

https://user-images.githubusercontent.com/32915560/187817707-dcb6ba6a-4793-4ad9-89ef-46761c86c6a4.mov

**Comment by Zane-Reisbig** [comment]
I am having the same problem on Windows. Very strange.

**Comment by yellowred** [comment]
Same issue in Zellij. `tmux` is faster, but still not as fast as plain terminal (I use alacritty). I run it on the top macbook pro m3.

**Comment by 2gn** [comment]
This discussion solved the issue for me https://github.com/wez/wezterm/discussions/2727

Add these lines in `wezterm.lua`

```wezterm.lua

return {
  animation_fps = 1,
  cursor_blink_rate = 0,
  cursor_blink_ease_in = "Constant",
  cursor_blink_ease_out = "Constant",
}
```
